web/client/epics/maps.js

```diff
@@ -300,7 +302,7 @@ const storeDetailsInfoEpic = (action$, store) =>
                 )
                     .switchMap((attributes) => {
                         let details = find(attributes, {name: 'details'});
-                        if (!details) {
+                        if (!details || details.value === EMPTY_RESOURCE_VALUE) {
```

Comment:
please add a test also when details is undefined (i.e. not present in the response )

Reply:
done